Source/Krypton Components/Krypton.Ribbon/Controls Ribbon/KryptonRibbonContext.cs

```diff
@@ -120,7 +120,7 @@ public Color ContextColor
             set
             {
                 // We never allow a null or transparent color
-                if ((value == null) || (value == Color.Transparent))
+                if (/*(value == null) ||*/ (value == Color.Transparent)) // 'color' is never 'null' CS8073
```

Comment:
**Do not do this** for public setters, the compiler can never know how these are going to be used in the "Wild"
Just add the warning to the `.editorconfig`

Reply:
Added to `.editorconfig`